app-cpp/app355.cpp

```diff
@@ -38,7 +38,7 @@ void SatOrbitSingle::OnCalculateOrbitalData(const std::vector<sat355::TLE>& inTL
 {   
     auto tleBegin = inTLEVector.begin();
     auto tleEnd = inTLEVector.end();
-    const auto size = std::distance(tleBegin, tleEnd);
+    const std::size_t size = static_cast<std::size_t>(std::distance(tleBegin, tleEnd));
```

Comment:
subjective:
the `static_cast` uses an explicit type, so you can use `auto` here:
```
    const auto size = static_cast<std::size_t>(std::distance(tleBegin, tleEnd));
```